src/transformStateWithClones.js

```diff
@@ -8,6 +8,29 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const result = [];
+  const copyState = { ...state };
+
+  for (const i in actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
i так себе название, если это не инкремент
используй for of для массивов